sapi/fpm/tests/gh12232-php-value-extension.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+FPM: gh12232 - loading shared ext in FPM config
+--SKIPIF--
+<?php
+include "skipif.inc";
+FPM\Tester::skipIfSharedExtensionNotFound('gnupg');
+?>
+--FILE--
+<?php
+
+require_once "tester.inc";
+
+$cfg = <<<EOT
+[global]
+error_log = {{FILE:LOG}}
+[unconfined]
+listen = {{ADDR}}
+pm = static
+pm.max_children = 1
+pm.status_path = /status
+catch_workers_output = yes
+php_admin_value[extension] = gnupg
```

Comment:
This test does not run in CI. Can we reproduce the issue with `dl_test`?

Reply:
Good point. I didn't know about dl_test so will look into it.

Reply:
Ok this took me a bit longer as it actually initially did not work due to different bug in ini registration that was incorrectly asserting that module needs to be temporary which is not the case in FPM case.

Reply:
Hopefully sorted now but this whole fpm extension loading is such a mess that there might still be some bugs. Think the whole concept is broken so not trying to make it all cleanly work - just the worst issues like crashes really. The real solution is to introduce a pool manger and load it normally sooner.